spec/dummy/app/views/home/forms.rb

```diff
@@ -71,7 +71,7 @@ def main
               include_blank: false,
               input_html: { 'data-width' => 'full' }
       end
-    }, sub: ''
+    }, sub: true
```

Comment:
Just curious: this wasn't working for me when I initially implemented it. Is there another change you made here outside this commit?